.github/workflows/compareNDandODbuilds.yml

```
@@ -2,24 +2,111 @@ name: Build and profile Android apps
 
 on:
   workflow_dispatch:
+    inputs:
+      PULL_REQUEST_NUMBER:
+        description: Pull request number to build the app from
+        required: false
+        default: ''
+      OLD_DOT_PULL_REQUEST_NUMBER:
+        description: Old Dot Pull request number to build the app from
+        required: false
+        default: ''
 
 jobs:
+  getBranchRef:
+    runs-on: ubuntu-latest
+    outputs:
+      REF: ${{ steps.getHeadRef.outputs.REF }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get ND PR commit SHA
```

I think we are mixing the ND and NewDot terms - would be good to just use one and NewDot is more explanatory

Same goes for OD/ OldDot
```suggestion
      - name: Get NewDot PR commit SHA
```

---

It's also a good point. I started to use shorthands but I agree that being verbose is better here